examples/eql/harmonic_spherical.py

```diff
@@ -0,0 +1,121 @@
+"""
+Gridding in Spherical coordinates
+=================================
+
+When gridding magnetic or gravity data on large regions thought the equivalent
+layer technique, the curvature of the Earth must be taken into account.
+Projecting data for large regions may introduce projection errors on the
+predictions.
+
+:class:`harmonica.EQLHarmonicSpherical` is able to estimate the
+coefficients of a set of point sources (the equivalent layer) that fit the
+observed harmonic data given in Spherical coordinates.
+It allows to benefit from the same advantages that the equivalent layer
+technique offers while taking into account the curvature of the Earth.
+"""
+import numpy as np
+import boule as bl
+import verde as vd
+import harmonica as hm
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+
+# Fetch the sample gravity data from South Africa
+data = hm.datasets.fetch_south_africa_gravity()
+
+# Downsample the number of points to speed-up the computations for this example
+reducer = vd.BlockMean(spacing=0.2, drop_coords=False)
+(longitude, latitude, elevation), gravity_data, _ = reducer.filter(
+    (data.longitude, data.latitude, data.elevation), data.gravity,
+)
```

Comment:
```suggestion
# Downsample the data using a blocked mean to speed-up the computations 
# for this example. This is preferred over simply discarding points to avoid 
# aliasing effects.
blocked_mean = vd.BlockReduce(np.mean, spacing=0.2, drop_coords=False)
(longitude, latitude, elevation), gravity_data = blocked_mean.filter(
    (data.longitude, data.latitude, data.elevation), data.gravity,
)
```

I usually go for `BlockReduce` in these cases to avoid having it return the weights (and the ugly `_` hack).